galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ICicsTerminal.java

```diff
@@ -28,4 +28,6 @@ public interface ICicsTerminal extends ITerminal {
      */
     public boolean isUppercaseTranslation() throws CicstsManagerException;
 
-}
\ No newline at end of file
+    String getLoginCredentials();
```

Comment:
This will obviously need to change if the change the attribute on the annotation

Reply:
Updated